packages/vue-components/lib/components/LayerListControls/LayerListControls.vue

```diff
@@ -58,6 +58,10 @@ export default {
     const { activeLegend, setActiveLegend } = useLegend(selectedIds)
     const { onSortingChange } = useSortable(layers, root, openItems)
 
+    onMounted(() => {
```

Comment:
You're delegating responsibility for clearing `state` (the `ref` in `useSelected`) to the Vue component. Is this really desired, or should  `useSelected()` return a 'scoped' piece of state, so that every `LayerListControls` component has its own `selectedIds` that is strictly private to that instance of the component?